plugins/outputs/postgresql/tables/table_manager.go

```diff
@@ -0,0 +1,245 @@
+package tables
```

Comment:
This file should be called `manager.go` to match the test file and because the containing package is already named tables.

I would reduce the number of packages personally, maybe just a single `schema` package in addition to the top package.